go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/cilium/coverbee v0.3.2
 	github.com/cilium/customvet v0.0.0-20221207181232-aa8731fa2d27
-	github.com/cilium/deepequal-gen v0.0.0-20200406125435-ad6a9003139e
+	github.com/cilium/deepequal-gen v0.0.0-20230330134849-754271daeec2
```

Comment:
upgraded manually in https://github.com/cilium/cilium/pull/24651

Reply:
yup, i noticed you have one pending Pr for this :+1: